web/ci.sh

```diff
@@ -24,15 +24,22 @@ cd "$THIS_SCRIPT_PATH"
 
 builder_describe "Defines and implements the CI build steps for Keyman Engine for Web (KMW)." \
   "build" \
```

Comment:
I'm not sure that we want `build` or `test` here as these are just wrappers for other scripts.

Reply:
I'd like to make an argument for `test`, at least.

It's been noted that the main `build.sh` scripts for each folder should really only run tests _for their folder_ and not for any dependencies.

Accordingly, this `ci.sh` action might be the perfect place to run _all_ the related tests, including those for dependencies.  We never did arrive at a good answer for this before, and I think it's at least a decent answer we can move forward with.

Reply:
I'm OK with this for now -- at least until we have a common/ build configuration for running tests.

Reply:
Refer to the unresolved comment under https://github.com/keymanapp/keyman/pull/7474#pullrequestreview-1198752695; this would probably be a "better" place to implement the `--all` bit to run all related unit tests.

I'm not saying it's the _best_ solution long-term, but it's quite approachable and low-cost to do with minimal changes elsewhere, letting us move forward on other things for now.